README.md

```diff
@@ -68,8 +68,15 @@ underscores is hard. So for example, `add_repo` can be typed as `addrepo`.
                                 name given in the second parameter. The location
                                 of the repo will be asked for and then added to
                                 your list of repos.
+* `repo_add_directory` or `rad` - Searches all subdirectories of the second parameter
+                                  and automatically adds each to the list of repos. This
+                                  command uses the relative path between the second parameter 
+                                  and the repo as the repo name. The second parameter can be
```

Comment:
Only issue I can see with that is what happens when we have duplicates? There were a couple  of generic repo names that would have duplicate entries if you ran it on the dudewheresmycode output folder. For example, the "dsi/root" and "dsi_api/root".

Maybe upon finding a duplicate it does the full path for the second item and beyond?